custom_components/ramses_cc/sensor.py

```diff
@@ -384,6 +384,9 @@ async def async_update(self) -> None:
         DEVICE_UNITS: UnitOfTemperature.CELSIUS,
     },
     SZ_FAN_INFO: {STATE_CLASS: None},
+    SZ_FILTER_REMAINING: {
```

Comment:
You need to import this from `ramses_rf.protcol.const`

Reply:
Yes, just realised that I forgot that. Thanks